src/main/java/com/iridium/iridiumskyblock/managers/IslandManager.java

```diff
@@ -53,13 +52,37 @@ public IslandManager() {
         super(IridiumSkyblock.getInstance());
     }
 
+    public boolean deleteWorld(File path) {
+        if(path.exists()) {
+            File files[] = path.listFiles();
+            for(int i=0; i<files.length; i++) {
+                if(files[i].isDirectory()) {
+                    deleteWorld(files[i]);
+                } else {
+                    files[i].delete();
+                }
+            }
+        }
+        return(path.delete());
+    }
```

Comment:
I can't believe there isn't really a better method for this in Java 8 without libraries... I suppose you could do this:

```suggestion
    public boolean deleteWorld(File path) {
        Files.walk(pathToBeDeleted)
            .sorted(Comparator.reverseOrder())
            .map(Path::toFile)
            .forEach(File::delete);
    }
```